app/controllers/support_rotations_controller.rb

```diff
@@ -60,11 +60,8 @@ def ops_eng_name
 
     def affected_projects
       in_hours_projects = (first_line_dev.projects + first_line_ops.projects).uniq
-      client_projects = in_hours_projects.select { |p| p.tenk_id != Project::TENK_ID_FOR_SUPPORT }
-      client_projects.select do |project|
-        start_date <= Date.parse(project.ends_at) &&
-          end_date >= Date.parse(project.starts_at)
```

Comment:
Why did this bit go away?

Reply:
Oh oops! That was from yet another issue, which I solved in the end by getting Farrah to update BEIS ODA's end date in 10kft 🙈 I'll bring it back.

Reply:
👍